ui/src/app/services/chrome-recoder.service.ts

```diff
@@ -75,8 +75,8 @@ export class ChromeRecorderService {
       this.messageEvent = event;
       if (this.isInstalledEvent) {
         this.isInstalled = true;
-      } else if (this.isStoppedRecorder) {
-        this.isStepRecording = false;
+      } else if (this.isStoppedRecorder && this.isStepRecorder) {
+        this.isStepRecorder = false;
```

Comment:
why do we need to toggle stepRecorder flag? what is the use case here? @manikanta-reddy-t

Reply:
i think we will enter into that block only when we stopped recording and if it is an step recorder. when we stopped recording then we will changing that to an false.